src/main/java/com/blackducksoftware/integration/hub/notification/NotificationViewResults.java

```diff
@@ -27,21 +27,19 @@
 import java.util.List;
 import java.util.Optional;
 
-import com.blackducksoftware.integration.hub.api.view.CommonNotificationState;
-
 public class NotificationViewResults {
```

Comment:
We may be able to kill this class if we move the fields into NotificationResults